src/Element/EntityAutocompleteMultiple.php

```diff
@@ -0,0 +1,277 @@
+    $button = $form_state->getTriggeringElement();
+
+    // Go one level up in the form, to the widgets container.
+    $element = NestedArray::getValue($form, array_slice($button['#array_parents'], 0, -1));
+    $field_name = $element['#field_name'];
+    $parents = $element['#parents'];
+
+    // Increment the items count.
+    $field_state = static::getWidgetState($parents, $field_name, $form_state);
+    $field_state['items_count']++;
+    static::setWidgetState($parents, $field_name, $form_state, $field_state);
+
+    $form_state->setRebuild();
+  }
+
+  /**
+   * Ajax callback for the "Add another item" button.
+   *
+   * @param array $form
+   *   The build form.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The form state.
+   *
+   * @return array|null
+   *   The element.
+   */
+  public static function addMoreAjax(array $form, FormStateInterface $form_state): ?array {
+    $button = $form_state->getTriggeringElement();
+    return NestedArray::getValue($form, array_slice($button['#array_parents'], 0, -1));
+  }
+
+  /**
+   * Helper method for extracting form element state data.
+   *
+   * @param array $parents
+   *   The parents array.
+   * @param string $field_name
+   *   The field name.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The form state.
+   *
+   * @return array|null
+   *   Actual form element state.
+   */
+  public static function getWidgetState(array $parents, string $field_name, FormStateInterface $form_state): ?array {
+    return NestedArray::getValue($form_state->getStorage(), static::getWidgetStateParents($parents, $field_name));
+  }
+
+  /**
+   * Helper method for updating form element state data.
+   *
+   * @param array $parents
+   *   The parents array.
+   * @param string $field_name
+   *   The field name.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The form state.
+   * @param array $field_state
+   *   The field state.
+   */
+  public static function setWidgetState(array $parents, string $field_name, FormStateInterface $form_state, array $field_state): void {
+    NestedArray::setValue($form_state->getStorage(), static::getWidgetStateParents($parents, $field_name), $field_state);
+  }
+
+  /**
+   * Returns the location of processing information within $form_state.
+   *
+   * @param array $parents
+   *   The array of #parents where the widget lives in the form.
+   * @param string $field_name
+   *   The field name.
+   *
+   * @return array
+   *   The location of processing information within $form_state.
+   */
+  protected static function getWidgetStateParents(array $parents, string $field_name): array {
+    // phpcs:disable
+    // Field processing data is placed at
+    // $form_state->get([
+    //   'field_storage',
+    //   '#parents',
+    //   ...$parents...,
+    //   '#fields',
+    //   $field_name,
+    // ]),
+    // to avoid clashes between field names and $parents parts.
```

Comment:
This function is a direct copy from the widgetBase one. Let's keep the same format.

Reply:
Returned but with temporary disabling phpsc.